llvm/lib/Transforms/Vectorize/VPlanHelpers.h

```diff
@@ -290,13 +290,6 @@ struct VPTransformState {
   /// vector loop.
```

Comment:
Is the above description still accurate? Would be good to clarify when State.addNewMetadata(To, Orig) needs to complement applyMetadata(To). Will possibly be handled by #136450

Reply:
I think this is still accurate; applyMetadata in this patch applies metadata already on `Orig`, while `addNewMetadata` adds metadata from versioning, not present at the original instructions.